x/poolmanager/create_pool.go

```diff
@@ -86,6 +86,11 @@ func (k Keeper) CreatePool(ctx sdk.Context, msg types.CreatePoolMsg) (uint64, er
 	}
 
 	emitCreatePoolEvents(ctx, poolId, msg)
+	// TODO: Remove gamm hooks and only use poolmanager hook for all types of poolType
+	// only call the hook through poolmanager if the pool created is CL
+	if msg.GetPoolType() == types.Concentrated {
+		k.hooks.AfterPoolCreated(ctx, sender, pool.GetId())
+	}
```

Comment:
Please make an issue for this

Reply:
https://github.com/osmosis-labs/osmosis/issues/4763